test/fixtures/cookbooks/test/recipes/client.rb

```diff
@@ -1,3 +1,3 @@
-execute 'apt-get update' if platform_family?('debian')
+apt_update 'update' if platform_family?('debian')
```

Comment:
the platform check can go away with new chef clients. I'm fairly certain we've fixed that finally